sys-kernel/installkernel/metadata.xml

```diff
@@ -12,6 +12,7 @@
 		<flag name="refind">Install a Gentoo icon for rEFInd alongside the (unified) kernel image, used icon is overridable with REFIND_ICON env var</flag>
 		<flag name="systemd">Use systemd's kernel-install to install kernels, overridable with SYSTEMD_KERNEL_INSTALL env var</flag>
 		<flag name="systemd-boot">Use systemd-boot's native layout by default</flag>
+		<flag name="ugrd">Generate an initrd using UGRD on each kernel installation</flag>
```

Comment:
Actually, now we are touching this file anyway, lets correct a mistake.

Here we say "initrd" whereas almost everywhere else we say "initramfs", lets make that consistent (both for the new ugrd flag and the dracut flag).

Reply:
I wish the use of "initramfs" were more consistent in general. It's a bit frustrating that installkernel and the kernel itself use "initrd" in arguments/config. Would there be any value in making it recognize config like "initramfs_generator" or would that just make things more confusing/hard to manage?